### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by jiaj12**
update zh content to match en master branch

### CODE REVIEW

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 1]
```diff
@@ -125,26 +125,32 @@ parameter.
...
 You can reference glossary terms with an inclusion that will automatically
 update and replace content with the relevant links from [our
 glossary](/docs/reference/glossary/). When the term is moused-over by someone
 using the online documentation, the glossary entry will display a tooltip.
+
+As well as inclusions with tooltips, you can reuse the definitions from the glossary in
+page content.
 -->
 ## 词汇
+两个词汇表提示
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:139]: 前面空行。后面标点。

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 2]
```diff
@@ -125,26 +125,32 @@ parameter.
...
 
 你可以通过加入术语词汇的短代码，来自动更新和替换相应链接中的内容
 （[我们的词汇库](/zh/docs/reference/glossary/)）
 这样，在浏览在线文档，鼠标移到术语上时，术语解释就会显示在提示框中。
 
+除了包含工具提示外，您还可以重用页面内容中词汇表中的定义。
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:145]: 不要使用您。

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 3]
```diff
@@ -154,9 +160,36 @@ For example, the following include within the markdown will render to
...
+
+```
+{{</* glossary_definition prepend="A cluster is" term_id="cluster" length="short" */>}}
+```
+
+<!--
+which renders as:
+{{< glossary_definition prepend="A cluster is" term_id="cluster" length="short" >}}
+-->
+呈现为： {{< glossary_definition prepend="A cluster is" term_id="cluster" length="short" >}}
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:179]: 请换行！！！

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 4]
```diff
@@ -154,9 +160,36 @@ For example, the following include within the markdown will render to
...
+
+<!--
+You can also include a full definition:
+-->
+您也可以包括完整的定义：
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:184]: 不要用您。
下面是代码段，请前后留空行。

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 5]
```diff
@@ -154,9 +160,36 @@ For example, the following include within the markdown will render to
...
+```
+{{</* glossary_definition term_id="cluster" length="all" */>}}
+```
+<!--
+which renders as:
+{{< glossary_definition term_id="cluster" length="all" >}}
+-->
+呈现为： {{< glossary_definition term_id="cluster" length="all" >}}
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:192]: 请换行。

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 6]
```diff
@@ -136,13 +136,14 @@ As well as inclusions with tooltips, you can reuse the definitions from the glos
 page content.
 -->
 ## 词汇
-两个词汇表提示
+
+两个词汇表提示。
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:140]:
```suggestion
有两种词汇表提示。
```

**Comment by tengqm** [comment]
pls squash your commits into one.

**Comment by jiaj12** [comment]
squashed

**Comment by jiaj12** [comment]
@markthink  need another reviewer

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 7]
```diff
@@ -199,7 +237,7 @@ Parameter | Description | Default
 {{< /table >}}
 -->
 
-```go-html-template
+​```go-html-template
```
Comment (howieyuen) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md:240]: what's the difference here?

Comment (jiaj12) [reply content/zh/docs/contribute/style/hugo-shortcodes/index.md:240]: 这有什么问题？

Comment (howieyuen) [reply content/zh/docs/contribute/style/hugo-shortcodes/index.md:240]: I don't understand why this line go changed

Comment (jiaj12) [reply content/zh/docs/contribute/style/hugo-shortcodes/index.md:240]: maybe it's a bug for github. 2 lines are exactly the same

Comment (howieyuen) [reply content/zh/docs/contribute/style/hugo-shortcodes/index.md:240]: I think maybe it is LF problem?

Comment (jiaj12) [reply content/zh/docs/contribute/style/hugo-shortcodes/index.md:239]: all \n in both files, no '\r\n's at all.

**Review by markthink — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**